deploy/charts/venafi-kubernetes-agent/README.md

```diff
@@ -0,0 +1,162 @@
+
+```shell
+export VENAFI_CLIENT_ID="<PASTE YOURS HERE>"
+kubectl create namespace ${VENAFI_NAMESPACE}
+kubectl create secret generic agent-credentials -n ${VENAFI_NAMESPACE} \
+  --from-file=privatekey.pem=${VENAFI_SERVICE_ACCOUNT}.pem
+```
+
+Install the chart by setting the `config.clientId` field:
+
+```shell
+helm upgrade --install agent deploy/charts/venafi-kubernetes-agent \
+  --namespace ${VENAFI_NAMESPACE} \
+  --set config.clientId="${VENAFI_CLIENT_ID}"
+```
+
+### 4) Add Cluster in Venafi Control Plane
+
+- Go to "Installations" -> "Kuberentes Clusters" [here](https://ui.venafi.cloud/clusters-inventory) and click "Connect"
+- On step 1 select "Continue".
+- On step 2 select "Advanced Connection".
+- On step 3 select "Continue" to skip.
+- On step 4, fill in the details as needed:
+  - "Name" should match your service account name from before, eg "example-cluster".
+  - Under "Service Account" click that drop down and select the previously created service account.
+  - Then check the "The connection command has completed." box and select "continue".
+- On step 5, either wait for validation or select "Finish" to go back to the cluster list.
+
+### 5) Deployment Verification
+
+Check the agent logs to ensure you see a similar entry to the following:
+
+```console
+2023/10/24 12:10:03 Running Agent...
+2023/10/24 12:10:03 Posting data to: https://api.venafi.cloud/
+2023/10/24 12:10:04 Data sent successfully.
+```
+
+You can do this with the following command:
+
+```shell
+kubectl logs -n ${VENAFI_NAMESPACE} $(kubectl get pod -n ${VENAFI_NAMESPACE} -l app.kubernetes.io/instance=agent -o jsonpath='{.items[0].metadata.name}')
+```
+
+You can also check inb the Venafi Control Plane to see when the "Last Check In" was for your cluster.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| authentication | object | `{"secretKey":"privatekey.pem","secretName":"agent-credentials"}` | Authentication details for the Venafi Kuberente Agent |
+| authentication.secretKey | string | `"privatekey.pem"` | Key name in the references secret |
+| authentication.secretName | string | `"agent-credentials"` | Name of the secret containing the privatekey |
+| command | list | `[]` | Specify the command to run overriding default |
+| config | object | `{"clientId":"","configmap":{"key":null,"name":null},"period":"0h1m0s","server":"https://api.venafi.cloud/"}` | Configuration section for the Venafi Kubernetes Agent itself |
+| config.configmap | object | `{"key":null,"name":null}` | Sepcify ConfigMap details to load config from an existing resource This should be blankby default unless you have you own config |
+| config.period | string | `"0h1m0s"` | Send data back to the platform every minute unless changed |
+| config.server | string | `"https://api.venafi.cloud/"` | Overrides the server if using a proxy in your environment |
```

Comment:
might be nice to also provide EU example in comment:
`for EU set to https://api.venafi.eu/ for example..`

Reply:
Good spot, I will do that.

Reply:
I couldn't see an easy way to get this injected into the table. But it is in the values.yaml as a comment.

Reply:
i don't see it in values.yaml sorry, can you link? i think also adding in README would be worth it

Reply:
it's here, but I spotted a typo in it anyway: https://github.com/jetstack/jetstack-secure/blob/de41a17fd9d6df7e8cdeee24c589fb9141543048/deploy/charts/venafi-kubernetes-agent/values.yaml#L88

Ah yes I can add it to the readme.

Reply:
i think the reamde section still has typos `environmen` and `varint`